### PR TITLE
[feature #158443085,#158422229] add sign up and login UI

### PR DESCRIPTION
**Pull request by TheSteelGuy**
**Introduction**
This PR creates sign up and sig in forms where users can sign up and sign in respectively
**Description of Task to be completed:
**
Sign up form with five fields for inputs
Login form with two input fields

**How to manually test**
```
Clone this repository
navigate into the repository after cloning
navigate further in the UI directory open the signup and login pages
double click index.html 
```
signin
signup

**Screenshots**
withbrowser

### CODE REVIEW

**Comment by larryTheGeek** [comment]
Clean design , good color combo

**Comment by Xerrex** [comment]
Good design

**Comment by Xerrex** [comment]
what is the purpose of the phone number ?

**Comment by AnthonyGW** [comment]
Please upload all the screenshots, so that we can see which images have been used. Include the browser window in the screenshots as well.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                <h4>SIGNUP <a href="UI/signup.html" style="text-decoration: none;">Here</a></h4>
+                <hr>
+            </div>
+        </div>
+        <div id="footer">
+            <p>&copy Ridemyway.All rights reserved </p>
+        </div>
+    </div><!--end of wrapper-->
+</body>
+</html>
```
Comment (AnthonyGW) [inline index.html]: Insert a new line at the end of the file

**Comment by TheSteelGuy** [comment]
This page has no picture as a background. For brower windo let me upload....

File: index.html (hunk 1 above)
Comment (TheSteelGuy) [reply index.html]: okay

Comment (AnthonyGW) [inline index.html]: Add a newline to the end of the file

**Pull request merged**